config/common.php

```diff
@@ -5,20 +5,10 @@
 use App\Contact\ContactMailer;
 use App\Service\Mailer;
 use Psr\Container\ContainerInterface;
-use Yiisoft\Aliases\Aliases;
 
 /* @var array $params */
 
 return [
-    ContainerInterface::class => static function (ContainerInterface $container) {
-        return $container;
-    },
-
-    Aliases::class => [
```

Comment:
That would remove aliases from console application and we want to have these.

Reply:
Maybe it makes sense to move the initialization of Aliases from https://github.com/yiisoft/yii-web/blob/master/config/web.php to https://github.com/yiisoft/aliases/blob/master/config/common.php?